components/src/components/Stack/Stack.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import {
+  Story,
+  Canvas,
+  Meta,
+  ArgsTable,
+  PRIMARY_STORY,
+} from '@storybook/addon-docs';
+import { VStack } from '.';
+import {
+  Source,
+  ComponentLinkRow,
+  SB_DESIGNSYSTEM_URL,
+  LinkToInteractiveStory,
+} from '../../storybook';
+
+<Meta title="Design system/Stack/API" component={VStack} />
+
+# VStack og HStack
+
+<ComponentLinkRow
+  // TODO: Oppdater med riktige lenker når designsystemet er publisert
```

Comment:
Denne kommentaren gjør at API-story ikke vises i stories under Stack. Jeg lagde en seksjon for komponenten i dokumentasjonen, dette er linken: https://design.domstol.no/987b33f71/p/8539cc-stack . Da trengs ikke to-do lenger 👍